### DISCUSSION TITLE
Ask for MFA code on every tsh login and web login

### DISCUSSION THREAD

**Discussion by AritraWork97** [gravitational/teleport#10908] [Q&A] [answered]
Hello everyone, 

I have google authenticator as the mfa of choise for teleport. I have enabled github oauth to login and everything works perfectly. Is there any way that on each login to any server, teleport asks for an MFA code from the google authenticator already being used to login.

Any help will be appreciated.

**Comment by scottpgallagher** [comment, marked as answer]
Yes you can do per session MFA on servers following this guide: https://goteleport.com/docs/access-controls/guides/per-session-mfa/
